### DISCUSSION TITLE
Male gender broken? Options say parent/mother, spouse/wife, child/daughter.

### DISCUSSION THREAD

**Discussion by Neriderc** [monicahq/monica#6150] [Q&A]
When choosing a relationship, most of the male options seem broken. It specifies parent/mother, spouse/wife, child/daugher rather than father/mother, husband/wife, son/daughter.

Some like uncle/aunt are working ok.

In personalisation settings the genders look like the default options. Rather not say/Other, Man/Male, Woman/Female.

Anyone know what I might have broken?

**Comment by peterleimbach** [comment]
I am new to monica but in my docker Installation with the version 3.7.0 I have the same issue.

The issue is in the english language of the UI as in the german though it seems to be not a translation topic for me.

Have you checked the existing issues if there is already an issue created?

**Reply by Neriderc** [reply]
I had tried and failed to find an issue about it, but it turns out this was resolved in a pull request a month ago - so I presume we are just waiting on the next release and it should be fixed.

See pull request #6039
